spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/order.rb'
+
+describe Order do
+  before 'each order do' do
+    subject.start_order(:hawaiian, 3)
+  end
+
+  it 'can i choose an item an store it?' do
+    expect(subject.current_order).to eq [{hawaiian: 7, item_no: 3, item_total: 21}]
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)